dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/columns/image-preview.test.js

```diff
@@ -46,5 +46,39 @@ define([
             });
 
         });
+
+        describe('handleKeyDown method', function () {
```

Comment:
It would be good also to test the case when the active element is input

Reply:
ok